subsys/bluetooth/controller/hci_internal.c

```diff
@@ -1653,6 +1657,12 @@ static uint8_t vs_cmd_put(uint8_t const *const cmd, uint8_t *const raw_event_out
 		return sdc_hci_cmd_vs_cis_subevent_length_set(
 			(sdc_hci_cmd_vs_cis_subevent_length_set_t const *)cmd_params);
 #endif
+
+#if defined(CONFIG_BT_OBSERVER) || defined(CONFIG_BT_CENTRAL)
+	case SDC_HCI_OPCODE_CMD_VS_SET_SCAN_CHANNEL_MAP:
+		return sdc_hci_cmd_vs_set_scan_channel_map(
+			(sdc_hci_cmd_vs_set_scan_channel_map_t const *)cmd_params);
+#endif
 	default:
```

Comment:
Now that you are anyways updating this code path, can you also add the handler for https://github.com/nrfconnect/sdk-nrfxlib/blob/main/softdevice_controller/include/sdc_hci_vs.h#L109?